vividus/src/main/resources/org/vividus/spring-configuration.xml

```diff
@@ -26,8 +21,8 @@
     <bean id ="stringEncryptor" class="org.jasypt.encryption.pbe.StandardPBEStringEncryptor">
         <property name="config">
             <bean class="org.jasypt.encryption.pbe.config.SimplePBEConfig">
-                <property name="algorithm" value="PBEWithMD5AndDES"/>
-                <property name="password" value="#{systemProperties['vividus.encryptor.password']?:'82=thuMUH@'}"/>
+                <property name="algorithm" value="#{T(org.vividus.configuration.ConfigurationResolver).DEFAULT_ENCRYPTOR_ALGORITHM}"/>
+                <property name="password" value="#{systemProperties['#{T(org.vividus.configuration.ConfigurationResolver).VIVIDUS_ENCRYPTOR_PASSWORD_PROPERTY}']?:'#{T(org.vividus.configuration.ConfigurationResolver).DEFAULT_ENCRYPTOR_PASSWORD}'}"/>
             </bean>
         </property>
     </bean>
```

Comment:
it should be created only once (in `ConfigurationResolver`) and then it should be re-used